src/browser/extensions/index.ts

```diff
@@ -7,27 +9,33 @@ let jupyterlab = [
     require("@jupyterlab/console-extension"),
     require("@jupyterlab/csvviewer-extension"),
     require("@jupyterlab/docmanager-extension"),
-    require("@jupyterlab/fileeditor-extension"),
     require("@jupyterlab/faq-extension"),
     require("@jupyterlab/filebrowser-extension"),
+    require("@jupyterlab/fileeditor-extension"),
     require("@jupyterlab/help-extension"),
     require("@jupyterlab/imageviewer-extension"),
     require("@jupyterlab/inspector-extension"),
     require("@jupyterlab/launcher-extension"),
+    require("@jupyterlab/mainmenu-extension"),
     require("@jupyterlab/markdownviewer-extension"),
+    require("@jupyterlab/mathjax2-extension"),
     require("@jupyterlab/notebook-extension"),
+    require("@jupyterlab/rendermime-extension"),
     require("@jupyterlab/running-extension"),
-    require("@jupyterlab/services-extension"),
     require("@jupyterlab/settingeditor-extension"),
     require("@jupyterlab/shortcuts-extension"),
     require("@jupyterlab/tabmanager-extension"),
     require("@jupyterlab/terminal-extension"),
+    require("@jupyterlab/theme-dark-extension"),
     require("@jupyterlab/theme-light-extension"),
     require("@jupyterlab/tooltip-extension")
 ];
 
 let mime = [
-    require("@jupyterlab/vega2-extension")
+    require("@jupyterlab/json-extension"),
+    require("@jupyterlab/pdf-extension"),
+    require("@jupyterlab/vdom-extension"),
+    // require("@jupyterlab/vega2-extension")
```

Comment:
I temporarily disabled it because it broke for soms reason I have not really investigated yet and then I kind of forgot. I'll make an atempt to fix it tomorrow or the day after.